addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -137,7 +137,9 @@ function factory(dependencies) {
             _newOrdered[index] = otherChatWindow.localId;
             _newOrdered[index + 1] = chatWindow.localId;
             this.update({ _ordered: _newOrdered });
-            chatWindow.focus();
+            if (!chatWindow.isFolded) {
+                chatWindow.focus();
+            }
```

Comment:
This fixes the issue but I don't think this is going to the root cause.
Prevented instruction is just setting `isDoFocus` to `true`.

Other ways of calling `focus` can lead to a crash as well. We need to decide what to do when focus is called on a chat window that is not visible at all or that is folded.